Core/Python/add_device_to_static_group.py

```diff
@@ -0,0 +1,173 @@
+#
+#  Python script using OME API to create a new static group
+#
+# _author_ = Grant Curell <grant_curell@dell.com>
+# _version_ = 0.1
+#
+# Copyright (c) 2020 Dell EMC Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+SYNOPSIS:
+   Add a host to an existing static group
+
+DESCRIPTION:
+   This script exercises the OME REST API to add a host to an
```

Comment:
Maybe rephrased to indicate adding one or more hosts given the device-name of the host
May help to add another mutually exclusive param for service tags instead - so the end user can pass in devicename / service-tag to find a match

Reply:
Working it now

Reply:
Functionality added as requested.